website/docs/cloud-docs/api-docs/assessment-results.mdx

```diff
@@ -0,0 +1,108 @@
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+# Assessment Results API
+
+An Assessment Result is the record of an instance of continuous validation such as [Drift Detection](/cloud-docs/workspaces/settings/drift-detection).
+
+## Show assessment result
+
+`GET api/v2/assessment-results/:assessment_result_id`
+
+| Parameter               | Description              |
+| ----------------------- | ------------------------ |
+| `:assessment_result_id` | The assessment result ID |
+
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/assessment-results/asmtres-cHh5777xm
+```
+
+### Sample Response
+
+```json
+{
+  "id": "asmtres-UG5rE9L1373hMYMA",
+  "type": "assessment-results",
+  "data": {
+    "attributes": {
+      "drifted": true,
+      "succeeded": true,
+      "error-msg": null,
+      "created-at": "2022-07-02T22:29:58+00:00",
+    },
+    "links": {
+      "self": "/api/v2/assessment-results/asmtres-UG5rE9L1373hMYMA/"
+      "json-output": "/api/v2/assessment-results/asmtres-UG5rE9L1373hMYMA/json-output"
+      "json-schema": "/api/v2/assessment-results/asmtres-UG5rE9L1373hMYMA/json-schema"
+      "log-output": "/api/v2/assessment-results/asmtres-UG5rE9L1373hMYMA/log-output"
+    }
+  }
+}
+```
+
+## Retrieve the JSON output from the assessment execution
+
+The following endpoints retrieve files documenting the plan, schema, and logged runtime associated with the specificied assessment result.
+The reponses do not adhrere to JSON API spec and are intended to provide complete context for the assessment result's otherwise summary level of information.
+
+`GET api/v2/assessment-results/:assessment_result_id/json-output`
+
+This endpoint returns the JSON plan output associated with the assessment result.
+
+`GET api/v2/assessment-results/:assessment_result_id/json-schema`
+
+This endpoint returns the JSON [provider schema](https://www.terraform.io/cli/commands/providers/schema) associated with the assessment result.
+
+`GET api/v2/assessment-results/assessment_result_id/log-output`
+
+This endpoint returns Terraform JSON log output.
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](/cloud-docs/users-teams-organizations/api-tokens#organization-api-tokens). You must access it with a [user token](/cloud-docs/users-teams-organizations/users#api-tokens) or [team token](/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens) that has admin level access to the workspace. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
```

Comment:
All of these, or just the last one? If it's all of them, let's move this up to be directly under the initial explanation and take it out of the notes text.

Reply:
Good call